composer.json

```diff
@@ -7,7 +7,10 @@
         "psr-4": {"Lullabot\\Drainpipe\\": "src/"}
     },
     "autoload-dev": {
-        "psr-4": {"Lullabot\\Drainpipe\\Tests\\Functional\\": "tests/src/functional/"}
+        "psr-4": {
+            "Lullabot\\Drainpipe\\Tests\\Functional\\": "tests/src/functional/",
+            "Lullabot\\Drainpipe\\Scaffold\\": "scaffold/"
```

Comment:
Is this line intentional? I don't see any PHP code in this directory.

Reply:
Let's remove this line and see if tests still pass.